tools/test_api.py

```diff
@@ -2274,20 +2274,21 @@ def test_spec_from_test_builds(test_builds):
     for build in test_builds:
         # Convert TZ target name to test spec platform name
         #
-        # 1. All TZ targets should have name pattern: PLATFORM_[PSA_]S/NS, where:
+        # 1. All TZ targets should have name pattern: PLATFORM_[PSA_/NPSA_]S/NS, where:
```

Comment:
@ccli8  Sorry for nitpicking. Having `PSA` in the target name is not mandatory as it is the default. I'd write this as
1. All TZ targets should have name pattern: PLATFORM_[NPSA_]S/NS, where:
(2) 'NPSA' for non-PSA targets. Defaults to PSA target if absent.
(3) 'S'/'NS' for secure/non-secure targets

Reply:
@ccli8 Once updated, leave a comment, we restart CI